regolith/helpers/a_expenseshelper.py

```diff
@@ -0,0 +1,91 @@
+"""Builder to add expenses """
+import datetime as dt
+import sys
+import time
+from argparse import RawTextHelpFormatter
+
+import nameparser
+
+from regolith.helpers.basehelper import SoutHelperBase, DbHelperBase
+from regolith.dates import month_to_int, month_to_str_int
+from regolith.fsclient import _id_key
+from regolith.sorters import position_key
+from regolith.tools import (
+    all_docs_from_collection,
+    filter_grants,
+    fuzzy_retrieval,
+)
+
+ALLOWED_TYPES = ["nsf", "doe", "other"]
+ALLOWED_STATI = ["invited", "accepted", "declined", "downloaded", "inprogress",
+                 "submitted", "cancelled"]
+
+
+def subparser(subpi):
+    subpi.add_argument("list_name", help="A short but unique name for the list",
+                        default=None)
+    subpi.add_argument("title", help="A title for the list that will be "
+                                     "rendered with the list")
+    subpi.add_argument("tags", help="The tags to use to build the list",
+                       nargs="+")
+    subpi.add_argument("-p", "--purpose",
+                        help="The purpose or intended use for the reading"
+                        )
+    return subpi
+
+class Expenses(DbHelperBase):
+    """Build a helper"""
+    btype = "a_expenses"
```

Comment:
let's keep it to only add one expense at a time, so it should be called ``a_expense``